cmd/config.go

```diff
@@ -0,0 +1,124 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/AlecAivazis/survey/v2/terminal"
+	"github.com/ory/viper"
+	"github.com/spf13/cobra"
+
+	bosonFunc "github.com/boson-project/func"
+)
+
+func init() {
+	root.AddCommand(configCmd)
+	configCmd.Flags().StringP("path", "p", cwd(), "Path to the project directory (Env: $FUNC_PATH)")
+}
+
+var configCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Configure a function",
+	Long: `Configure a function
+
+Interactive propmt that allows configuration of Volume mounts and Environment variables for a function
+project present in the current directory or from the directory specified with --path.
+`,
+	SuggestFor: []string{"cfg", "cofnig"},
+	PreRunE:    bindEnv("path"),
+	RunE:       runConfigCmd,
+}
+
+func runConfigCmd(cmd *cobra.Command, args []string) (err error) {
+
+	function, err := initConfigCommand(args)
+	if err != nil {
+		return
+	}
+
+	var qs = []*survey.Question{
+		{
+			Name: "selectedConfig",
+			Prompt: &survey.Select{
+				Message: "What do you want to configure?",
+				Options: []string{"Environment values", "Volumes"},
+				Default: "Environment values",
+			},
+		},
+		{
+			Name: "selectedOperation",
+			Prompt: &survey.Select{
+				Message: "What operation do you want to perform?",
+				Options: []string{"Add", "Remove", "List"},
+				Default: "List",
+			},
+		},
+	}
+
+	answers := struct {
+		SelectedConfig    string
+		SelectedOperation string
+	}{}
+
+	err = survey.Ask(qs, &answers)
+	if err != nil {
+		if err == terminal.InterruptErr {
+			return nil
+		}
+		return
+	}
+
+	switch answers.SelectedOperation {
+	case "Add":
+		if answers.SelectedConfig == "Volumes" {
+			err = runAddVolumesPrompt(cmd.Context(), function)
+		}
+	case "Remove":
+		if answers.SelectedConfig == "Volumes" {
+			err = runRemoveVolumesPrompt(function)
+		}
+	case "List":
+		if answers.SelectedConfig == "Volumes" {
+			listVolumes(function)
+		}
+	}
+
+	return
+}
+
+// CLI Configuration (parameters)
+// ------------------------------
+
+type configCmdConfig struct {
+	Name    string
+	Path    string
+	Verbose bool
+}
+
+func newConfigCmdConfig(args []string) configCmdConfig {
```

Comment:
Dangerously close to `AbstractConfigServletInitializerConfig` :rofl:

Reply:
Haha right 😅 I just wanted to follow the existing patterns for naming...